### PR TITLE
Gives AIs two HUD buttons (Manifest + Overwatch), some QoL and removes multicam mode

### LINKED ISSUE #10771: Widescreen leaves some AI icons floating in the wrong place.
### Description

The title and the attached picture describe it the best, i believe. 

### Test Merges

Server-Side Code Modifications: https://tgstation13.org/parsed-logs/sybil/server_side_modifications.dm

Round ID: [16023](https://scrubby.melonmesa.com/round/16023) - THIS IS WRONG IT WAS THE PREVIOUS ROUND SO I GUESS IT IS 16022
BYOND Version: 514.1584
Server revision compiled on: 
Master commit: [599c26055d187dd3964ca10f039365268e70eaee](https://github.com/tgstation/TerraGov-Marine-Corps//commit/599c26055d187dd3964ca10f039365268e70eaee)
The following pull requests are currently test merged:
[#10762: 'Revert "Improve do_surgery code"' by BraveMole at commit 5f524fc9f8](https://github.com/tgstation/TerraGov-Marine-Corps//pull/10762)
[#10750: 'Projectile performance improvement' by BraveMole at commit baeb9395d6](https://github.com/tgstation/TerraGov-Marine-Corps//pull/10750)
[#10669: 'Add a rust pathfinding' by BraveMole at commit 8306df43a9](https://github.com/tgstation/TerraGov-Marine-Corps//pull/10669)
[#10646: 'Combat Patrol Base Remap and Fluff' by tyeagg at commit 9363ed1b8b](https://github.com/tgstation/TerraGov-Marine-Corps//pull/10646)

Local commit: 543f2bd36ff3a71a0b152d298df906eff86fb2d6
Server tools version: /tg/station 13 Server v3.2.6.2

### Reproduction Steps

1. Play AI on widescreen.
2. See multicamera icon floating.
3. Cry.

### Screenshots

tgmc 2022-08-15 184040

### PR DESCRIPTION
**Pull request by mc-oofert**
## About The Pull Request

2023-04-11 16_47_49-Window
Gives AI two new buttons in the HUD:
View Crew Manifest (self-explanatory, faster than verb tab)
Main Overwatch quick-access (faster than moving your camera to CIC while you were watching planetside stuff)
Main Overwatch button does not function if the Main Overwatch console is gone or destroyed.

"NOTICE- x taking off towards y" shuttle transit notifications now mostly display the shuttles name instead of ID, was unsure if i should remove "Transit towards id/name" (also added better names for some shuttles)
HUD buttons moved
Eyeobj alert removed because AIs have no clue what an eyeobj is moreso what byond code is (also makes no sense IC)
REMOVES MULTI-CAMERA MODE.

Ports camera hotkeys to replace multicam mode from tgstation
> As the AI, you can hold CTRL and use numbers 1 to 9, to set a "hotkey" for that camera spot. Using it without CTRL brings you > to that saved spot, if any. 0 brings you back to the camera spot before you used it.

COMSIG_GLOB_SHUTTLE_TAKEOFF now gives you the mobile dock (which you can access its ID) instead of the mobile docks ID.
## Why It's Good For The Game

Multicam mode removed because it took me 5 seconds on local to break the thing and there isnt any real use for it in TGMC for AIs.
HUD button layout change has been changed to cameras on the left and info/announcements on the right
View Crew Manifest button is faster than finding it in the verb tab
Main Overwatch button is faster than moving your eye to CIC after watching marines groundside
fixes #10771
## Changelog
:cl:
add: Main Overwatch quick-access HUD button for AIs
add: Crew Manifest HUD button for AIs
del: Removed multicam mode from AIs
qol: AI shuttle movement alerts now use the shuttles name instead of ID (mostly)
/:cl:

### CODE REVIEW

**Comment by Clay500z** [comment]
I don't know about removing multi cam entirely, I do use it when I play as AI. It's good for saving locations that I want to jump back to. At round start I'll set a cam at CIC, req, and fob. Sometimes I'll set temporary cams at disks so I can jump back and forth between watching fob for xenos and watching disks. Though yeah, it's extremely buggy. Trying to use it for anything other than clicking once on a cam to jump to the location seems to break it for the rest of the round.

**Comment by bowlofcereal** [comment]
> I don't know about removing multi cam entirely, I do use it when I play as AI. It's good for saving locations that I want to jump back to. At round start I'll set a cam at CIC, req, and fob. Sometimes I'll set temporary cams at disks so I can jump back and forth between watching fob for xenos and watching disks. Though yeah, it's extremely buggy. Trying to use it for anything other than clicking once on a cam to jump to the location seems to break it for the rest of the round.

Im honestly impressed because I have played hundreds of hours of AI across SS13 yet have almost never touched on multicam. Ive been aware of its presence but I dont use it. I dont think many people everywhere do actually use it, its like this mechanic that we know exists but we dont use. I dont see much use for it specifically here unlike normal SS13 though, so seeing it go (especially if it causes bugs), isnt really a bad thing

**Review by QualityVan — CHANGES_REQUESTED** [review]
Map stuff is just multicam zoning things, yeah?

File: code/modules/mob/living/silicon/ai/freelook/chunk.dm [hunk 1]
```diff
@@ -84,7 +84,7 @@
 		added_turf.vis_contents -= vis_contents_opaque
 
 	for(var/turf/removed_turf AS in visRemoved)
-		if(obscuredTurfs[removed_turf] && !istype(removed_turf, /turf/open/ai_visible))
+		if(obscuredTurfs[removed_turf])
```
[the remark this answers is not in the document]
Comment (mc-oofert) [reply code/modules/mob/living/silicon/ai/freelook/chunk.dm:87]: ai_visible turf is for multicam

File: code/_onclick/hud/ai.dm [hunk 2]
```diff
@@ -80,33 +80,40 @@
...
+	name = "View Crew Manifest"
+	icon_state = "manifest"
 
-/atom/movable/screen/ai/multicam
-	name = "Multicamera Mode"
-	icon_state = "multicam"
 
-
-/atom/movable/screen/ai/multicam/Click()
+/atom/movable/screen/ai/view_manifest/Click()
```
[the remark this answers is not in the document]
Comment (mc-oofert) [reply code/_onclick/hud/ai.dm:88]: yeah should try that himme a min

File: code/modules/keybindings/bindings_client.dm [hunk 3]
```diff
@@ -108,3 +112,7 @@
 	for(var/key in keys_held)
 		keyUp(key)
 
+/datum/proc/key_down(key, client/user) // Called when a key is pressed down initially
+	return
+/datum/proc/key_up(key, client/user) // Called when a key is released
+	return
```
Comment (QualityVan) [inline code/modules/keybindings/bindings_client.dm:118]: whoops forgot to save this one.
This is an awful lot of extra calls for what's 99% of the time a noop. Could you make ai hotcams normal hotkeys instead?

File: code/_onclick/hud/ai.dm (hunk 2 above)
[the remark this answers is not in the document]
Comment (mc-oofert) [reply code/_onclick/hud/ai.dm:88]: it dont

[the remark this answers is not in the document]
Comment (mc-oofert) [reply code/_onclick/hud/ai.dm:88]: got it

File: code/modules/keybindings/bindings_client.dm (hunk 3 above)
Comment (mc-oofert) [reply code/modules/keybindings/bindings_client.dm:118]: i would need like 40 hotkeys

**Pull request closed without merging**